main.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+// I will import the package fmt and in the future a json
+
+import "fmt"
+
+func main() {
+	contactList := generateContacts()
+	showContacts(contactList)
+}
+
+//I tried to use iota to create a estructure type such as a enum, but I changed my mind for now
+
+/*
+type SeniorityType int
+
+ const(
+
+	entryLevel= iota +1
+	softwareEngineerI
+	softwareEngineerII
+	softwareEngineerIII
+	technicalLead
+	projectLead
+	manager
+	vp
+	director
+	cto
+
+ )
+*/
+
+//Defined a type of ths date structure person
```

Comment:
You have a typo here
```suggestion
//Defined a type of this date structure person
```

Reply:
tks! changed!